prac_05/emails.py

```diff
@@ -0,0 +1,28 @@
+def main():
+    emails_dic = {}
+    while True:
+        email = input("Email: ")
+        if email == "":
+            break
+
+        name = extract_name_from_email(email)
+        confirm_name = input(f"Is your name {name}? (Y/n) ").strip()
+
+        if confirm_name.lower() == 'n':
+            name = input("Name: ").strip()
+
+        emails_dic[email] = name
+
+    for email, name in emails_dic.items():
+        print(f"{name} ({email})")
+
+
+def extract_name_from_email(email):
+    name = email.split('@')[0]
+    name_components = name.split('.')
+    formatted_name = ' '.join(name_component.title() for name_component in name_components)
+    return formatted_name
+
```

Comment:
After the user finishes inputting email addresses and names, the code iterates over the emails_dic dictionary using a for loop.
For each key-value pair (email, name) in the dictionary, it prints the name followed by the email address in parentheses.